ui/js/component/transactionList/internal/TransactionListItem.jsx

```diff
@@ -6,12 +6,59 @@ import Link from "component/link";
 import lbryuri from "lbryuri";
 
 class TransactionListItem extends React.PureComponent {
+  abandonClaim() {
+    const {
+      claim_id: claimId,
+      claim_name: name,
+      txid,
+      type,
+      nout,
+    } = this.props.transaction;
+    let msg;
+
+    if (type == "tip") {
```

Comment:
"tip" ought to be a constant so that it can be clear where it is being referenced. Probably via a new file in the `constants` folder, since it doesn't fit in clearly anywhere.

Reply:
Done.